crates/algokit_utils/src/clients/app_manager.rs

```diff
@@ -311,35 +311,7 @@ impl AppManager {
         Ok(values)
     }
 
-    /// Decode box value using ABI type.
-    pub async fn get_box_value_from_abi_type(
```

Comment:
We should still have these functions, getting a box value using an AbiMethod doesn't make sense.